c/public/lib/source/cffread/cffread.c

```diff
@@ -974,15 +974,16 @@ static void saveBlend(cfrCtx h, float *realValue, abfOpEntry *blendEntry) {
         // copy the defaul region value
         blendValues[0] = defaultValue = INDEX_REAL(0);
 
-        /* now, copy the blend abusolate values to blendArray[i].
+        /* now, copy the blend absolate values to blendArray[i].
```

Comment:
Did you mean `absolute`?

(Otherwise these changes looks fine to me.)

Reply:
Ah I didn't realize there are two typos in one word... will fix